config/crd/bases/multicluster.openshift.io_multiclusterengines.yaml

```diff
@@ -28,8 +28,11 @@ spec:
     name: v1
     schema:
       openAPIV3Schema:
-        description: MultiClusterEngine is the Schema for the multiclusterengines
-          API
+        description: MultiClusterEngine defines the configuration for an instance
+          of a multicluster engine, a central  point providing the foundational components
+          for managing multiple Kubernetes-based clusters. The deployment of the components
+          of the multiclsuterengine will be determined based on the configuration
+          defined in this resource.
         properties:
```

Comment:
```
 MultiClusterEngine defines the configuration for an instance
          of a multicluster engine, a central point providing the foundational components
          for managing multiple Kubernetes-based clusters. The deployment of the multicluster engine components is 
          determined based on the configuration that is defined in this resource.
```

Reply:
I took out an extra space.
I made the last line more active and concise.
Added relative pronoun.
Fixed one type, but is this also a typo: defiMulticlusterHub
I could not edit directly, so please feel free to copy this after https://github.com/stolostron/multiclusterhub-operator/pull/4 is answered. @cameronmwall